### PR TITLE
nexctl: Move connections command under nexd

### PR DESCRIPTION
**Pull request by russellb**
Rename `nexctl connections` to `nexctl nexd connections`.

Everything else that interacts with `nexd` is under that subcommand.
Other subcommands relate to resources in the Nexodus API.

Signed-off-by: Russell Bryant <rbryant@redhat.com>

### CODE REVIEW

**Comment by russellb** [comment]
Conflicts with #940, so marking this one as a draft until that one merges

**Comment by nerdalert** [comment]
@russellb On your rebase, mind adding it to `ops/ansible/aws/validate-connectivity/tasks/main.yml` as well please sir:
```
    nexctl connections
    nexctl connections >> connectivity-results.txt 2>&1
```

**Review by nerdalert — APPROVED** [review]
LGTM when ready

**Pull request merged**
